build.gradle

```diff
@@ -333,7 +333,7 @@ task cmakeJniLib(type:Exec) {
     workingDir 'jni'
     def args = []
     args.add("cmake")
-    args.add(".")
```

Comment:
Good update. This makes cleaning some much easier. That being said, in the buildJniLib, instead of changing workingDir, can we just do:
```
make -Cbuild
```

I worry if we add "/" we might end up having issues with path resolution on different platforms (unless gradle handles this for us)

Reply:
The windows build seems to work fine https://github.com/opensearch-project/k-NN/actions/runs/13020810027/job/36325044749?pr=2442

Reply:
Sure - think thats fine